tests/pkcli/ci_test.py

```diff
@@ -4,25 +4,24 @@
 :copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
+from pykern import pkunit
```

Comment:
In pytests, it is important to not import anything globally, except pytest. Move to the test methods.

Reply:
a9e80a0beda32d4163def682c4cffe18637e6076